Automattic/bugomattic
# Redesign Next Steps

#### What Does This PR Add/Change?

Context: pciE2j-1VQ-p2#comment-1674. 

Links on the title are removed. A CTA button is added for tasks with links instead. 

##### BEFORE
![before](https://user-images.githubusercontent.com/67279475/232012302-4c6c5193-3825-4127-ad20-e49b37c9152c.png)

##### AFTER
**Task 1: Long title with link. Task 2: Long details without link**

![Screenshot 2023-04-14 at 16 42 50](https://user-images.githubusercontent.com/67279475/232076605-c4772b9e-3e73-4e67-a168-3cdbd9db370f.png)

**All tasks with links**
 
![Screenshot 2023-04-14 at 14 19 57](https://user-images.githubusercontent.com/67279475/232041701-35567231-c450-4f16-944e-7f3a0913a2df.png)


#### Testing Instructions



Working on CSS is another level 😅 I tried to fix all the possible scenarios! Tests are already updated according to the changes.

- Make sure that when clicking the button, you are redirected to the right page. 
- The trigger to checkbox change is still there: make sure that the checkbox is also checked when clicking the button.

**axe DevTools**
_This will be my first time using this extension._

- [Test URL](http://localhost:3000/Automattic/bugomattic?activePage=reportingFlow&duplicateSearch%5BsearchTerm%5D=&duplicateSearch%5BstatusFilter%5D=all&duplicateSearch%5Bsort%5D=relevance&issueDetails%5BfeatureId%5D=Product__Feature%20Group__Feature%20%28Under%20Feature%20Group%29&issueDetails%5BissueType%5D=bug&issueDetails%5BissueTitle%5D=&activeReportingStep=nextSteps)
- No automatic issues found 

#### Issues



Related to #  
Closes #53
